src/web/components/TeamMember/TeamMembersTable.tsx

```diff
@@ -37,6 +42,16 @@ function TeamMembersTableContent({
   const { sortData } = useSortable<UserDTO>();
   const sortedTeamMembers = sortData(teamMembers);
 
+  useEffect(() => {
+    const user = teamMembers.find((teamMember) => teamMember.id === LoggedInUser?.user?.id);
```

Comment:
there's a case where this logic won't work where the user is a UID2 support user but not actually a team member for the given participant. Hopefully this is an easy fix where we also just check the user directly for the UID2 support role, i.e. something like:
```
useEffect(() => {
  const user = teamMembers.find((teamMember) => teamMember.id === LoggedInUser?.user?.id);
  const isAdminUser = user?.currentParticipantUserRoles?.find(
    (role) => role.id === UserRoleId.UID2Support || role.id === UserRoleId.Admin
  );
  if (isAdminUser || LoggedInUser?.user.isUid2Support) {
    setAllowTeamMemberActions(true);
  }
}, [LoggedInUser, teamMembers]);
```
(once we fix up the type of the LoggedInUser to include `isUid2Support`.)

Reply:
also, I'm guessing you did this for a good reason, but it seems a bit strange to iterate through all the team members here when we already know who the user is. I'm guessing it's because the `LoggedInUser` does not have `currentParticipantUserRoles` whereas the list of `teamMembers` does. Per your comment below, it would be nice if `LoggedInUser` had all the user role stuff we needed so we didn't need to iterate through `teamMembers` here.